package.json

```diff
@@ -27,10 +27,10 @@
     "prebuild": "npm run clean",
     "prepublish": "npm run build",
     "release": "standard-version",
-    "security": "nsp check",
+    "security": "npm audit --json --audit-level high | npm-audit-helper --prod-only",
     "test": "jest",
     "test:watch": "jest --watch",
-    "test:coverage": "jest --collectCoverageFrom='src/**/*.js' --coverage",
+    "test:coverage": "jest --collectCoverageFrom=src/**/*.js --coverage",
```

Comment:
Use `"` quotes otherwise it will not work on windows

Reply:
Actually I removed it because it didn't work on windows

Reply:
strange, `"` should works on windows, do you use cmd or powershell?

Reply:
Original command on Windows
![image](https://user-images.githubusercontent.com/7988466/69063180-5767bd80-0a1c-11ea-84eb-0a8fc6ca2ad0.png)


After removing quotes
![image](https://user-images.githubusercontent.com/7988466/69063121-399a5880-0a1c-11ea-9c22-b27ae98de64e.png)

Reply:
Can you replace `'` on `"`?

Reply:
Works well with `"`. I have updated it.